presto-native-execution/src/test/java/com/facebook/presto/spark/TestPrestoSparkNativeExecution.java

```diff
@@ -84,7 +86,14 @@ protected QueryRunner createQueryRunner()
 
         String dataDirectory = System.getProperty("DATA_DIR");
 
-        return PrestoSparkQueryRunner.createHivePrestoSparkQueryRunner(configs, Optional.ofNullable(dataDirectory).map(Paths::get));
+        return PrestoSparkQueryRunner.createHivePrestoSparkQueryRunner(
+                ImmutableSet.of(
```

Comment:
Why not use io.airlift.tpch.TpchTable#getTables like before?

Reply:
`getTables()` would load all tables.

Reply:
Sure, but why not?

Reply:
Each table, takes up time to load. Slowing down the time taken to start execution of the actual test ?

Reply:
I suggest to use getTables, but if not, use io.airlift.tpch.TpchTable#ORDERS and similar to avoid hard-coding table names.

Reply:
> Each table, takes up time to load. Slowing down the time taken to start execution of the actual test ?

Just use -DDATA_DIR.

Reply:
Ah I see. Ok. I will stick to `getTables()` . Updated